packages/plugins/plugin-table/src/components/ColumnActionsMenu.tsx

```diff
@@ -18,7 +18,7 @@ type ColumnActionsMenuProps = {
   onOpenChange: (open: boolean) => void;
 };
 
-export const ColumnActionsMenu: React.FC<ColumnActionsMenuProps> = ({ tableModel, columnId, open, onOpenChange }) => {
+export const ColumnActionsMenu = ({ tableModel, columnId, open, onOpenChange }: ColumnActionsMenuProps) => {
```

Comment:
Thanks for this — I think this yields less lossy typing (should the type of `FC` or the implied type of what this returns ever change) and feels more intuitive to me.